packages/components/src/popover/index.js

```diff
@@ -80,21 +81,28 @@ class Popover extends Component {
 	}
 
 	componentWillUnmount() {
-		this.toggleWindowEvents( false );
+		this.toggleEvents( false );
 
 		clearTimeout( this.focusTimeout );
 	}
 
-	toggleWindowEvents( isListening ) {
+	toggleEvents( isListening ) {
 		const handler = isListening ? 'addEventListener' : 'removeEventListener';
 
 		window.cancelAnimationFrame( this.rafHandle );
 		window[ handler ]( 'resize', this.throttledComputePopoverPosition );
 		window[ handler ]( 'scroll', this.throttledComputePopoverPosition, true );
+
+		const hookNamespace = uniqueId( 'core/components/popover/toggle-events-' );
```

Comment:
I'm not very familiar with how hooks work, but I'm wondering if this works as intended.
If I'm not mistaken, we're creating a namespace when we mount the component and a different one when we unmount it.
In other words:
- On mount: `addAction( 'foo', 'bar-1', () => {} )`
- On unmount: `removeAction( 'foo', 'bar-2' )`

This probably is not a big deal anyway, but I've tried moving the definition in the constructor, changing it into:
```es6
this.hookNamespace = uniqueId( 'core/components/popover/toggle-events-' );
```
and it seems to keep working in the same way. 🤔

Reply:
You're right! Probably it's not needed to remove the action hook for having this working. Anyway, it makes sense to use the same namespace for adding/removing the same hook so I moved it to the constructor as suggested.